task.go

```diff
@@ -415,8 +415,14 @@ func (e *Executor) runCommand(ctx context.Context, t *taskfile.Task, call taskfi
 			return nil
 		}
 
-		stdOut := e.Output.WrapWriter(e.Stdout, t.Prefix)
-		stdErr := e.Output.WrapWriter(e.Stderr, t.Prefix)
+		var stdOut, stdErr io.Writer
+		if t.Interactive {
+			stdOut = output.Interleaved{}.WrapWriter(e.Stdout, t.Prefix)
+			stdErr = output.Interleaved{}.WrapWriter(e.Stderr, t.Prefix)
+		} else {
+			stdOut = e.Output.WrapWriter(e.Stdout, t.Prefix)
+			stdErr = e.Output.WrapWriter(e.Stderr, t.Prefix)
+		}
```

Comment:
```suggestion
		stdOut := e.Output.WrapWriter(e.Stdout, t.Prefix)
		stdErr := e.Output.WrapWriter(e.Stderr, t.Prefix)

		if t.Interactive {
			stdOut = output.Interleaved{}.WrapWriter(e.Stdout, t.Prefix)
			stdErr = output.Interleaved{}.WrapWriter(e.Stderr, t.Prefix)
		} 
```

Reduce readability complexity and same think for Go compiler.

Reply:
Your're right.  Funnily enough this is how my first patch looked like.
It will be changed back :+1:

Reply:
Great! If I can help 🤗 

Thanks for this PR!